test/fixtures/v5/roundabout_turn/sharp_left_default.json

```diff
@@ -9,7 +9,7 @@
     },
     "instructions": {
         "ar": " اسلك مسار الدخول الى الدوران",
-        "da": "Foretag et skarpt venstresving",
+        "da": "Drej til skarpt venstresving",
```

Comment:
Should be "Drej skarpt til venstre"

Reply:
And here "Foretag et" is supposed to use with "straight" modifier only? I also recommend to use most common phrase beginning.

Reply:
No, for `straight` you would say "Fortsæt ligeud" (Continue straight).

"Foretag et" is used for U-turns.